lib/libsys/fork.2

```diff
@@ -68,6 +68,15 @@ by the parent.
 This descriptor copying is also used by the shell to
 establish standard input and output for newly created processes
 as well as to set up pipes.
+Any file descriptors that were marked with the close-on-fork flag,
+.Dv FD_CLOFORK
+(see
+.Fn fcntl 2
+and
+.Dv O_CLOFORK
+in
+.Fn open 2
+), will not be present in the child process, but remain open in the parent.
```

Comment:
Are you sure this renders properly? I'd expect the close-paren to be on the previous like, i.e., `.Fn open 2 ),`